arrocke/gloss-translation
# feat: configure default reference translations for a language

## What has changed


Added a Bible translation selector to the manage language page.

## Connected Issues



Closes #171 

## QA Steps
1. Go to the manage language page for a language
2. Click on the bible translation dropdown
3. Select a few translations
4. Click the "submit" button
5. Refresh the browser page
    - The selects translations should persist



## Post-Deployment




## Saved for Future Development
1. Allow user to re-order selected translations
    - #208
7. Allow user to filter translation options list
8. Display completion percentage for a given translation (I figured this would fit better with the UI for point 1)
    - #211
